job-roles/README.md

```diff
@@ -0,0 +1,20 @@
+# Roles at MayaData
+
+{{#include intro.md}}
+
+## Typicall Engineering Team
```

Comment:
Spellcheck! Typical

Reply:
:facepalm:

Reply:
Thank you. Done some other spell checking as well.